.auri/$ppl8db3v.md

```diff
@@ -0,0 +1,6 @@
+---
+package: "@lucia-auth/adapter-postgresql" # package name
+type: "patch"
```

Comment:
```suggestion
type: "minor"
```

Reply:
Is this a breaking change?  `CONTRIBUTING.md` says `type should be minor if it introduces breaking change or patch if not`.  It's new functionality?

Reply:
Done.